Sources/Protocols/MessagesDataSource.swift

```diff
@@ -102,6 +102,17 @@ public protocol MessagesDataSource: AnyObject {
     ///
     /// The default value returned by this method is `nil`.
     func messageBottomLabelAttributedText(for message: MessageType, at indexPath: IndexPath) -> NSAttributedString?
+
+    /// The attributed text to be used for cell's timestamp label.
```

Comment:
`The timestamp label is shown when `showMessageTimestampOnSwipeLeft` is enabled by swiping left over the chat controller` - can we add this? Or how users know what is timestamp label? :)